source/reference/upgrade.txt

```diff
@@ -132,7 +132,8 @@ Version 6.0 Breaking Changes
   callback returns. In previous driver versions, this method
   returns the server command response, which varies depending on the MongoDB
   Server version or type that the driver connects to. To learn more
-  about transactions, see the :ref:`node-usage-txns` usage
+  about transactions, see the :ref:`Convenient Transaction API
+  <node-usage-convenient-txn>` and :ref:`Core API <node-usage-core-txn>` usage
```

Comment:
```suggestion
  <node-usage-convenient-txn>` and :ref:`Core API <node-usage-core-txn>`
```
S: can't delete examples on the next line, but can you change to say 'guides' instead?